### ISSUE TITLE
Better default behavior for appserver?

### ISSUE THREAD

**Issue by jamescheney** [links-lang/links#153]
The appserver is great because it means we can set multiple entry points into an application. This is just what you want for complex applications, but for simple ones (e.g. the vast majority of our toy examples) it necessitates some "boilerplate" at the end of the Links program that looks like this:

```
fun mainPage(_) {
  page
    ... display the actual main page ...
}

fun main() {
  addRoute("",mainPage);
  servePages()
}

main()
```

This means in particular that the naive exercise of copying an "example program" off the Links demo programs page and running "links" on it yields unhelpful results along the lines of:
```
(0, fun, fun) : Page
```

Since I need to teach some people how to do simple things with Links tomorrow, this boilerplate saddens me.

One way this could be nicer is if it were possible to call addRoute/servePages from the toplevel, i.e. inline the call to main().  Another even nicer thing would be if running `linx` (possibly with some to-be-defined flag) on a program whose final action is to construct a page were to result in starting the appserver with that page as the main page.  Is there an obvious obstacle to this solution?

**Comment by jgbm** [comment]
Some discussion resulted in suggesting that we could have a couple of convenience functions, such as:

| Function | Description |
|---|----|
| ```serveThis(page)``` | Starts the server, serving a single page |
| ```serveThese(pageList)``` | Starts the server with a list of routes and pages (or page-generating functions) |
|---|---|

These should be mostly one-liners in terms of the existing functions.

**Comment by jamescheney** [comment]
yup, will do.

**Comment by jamescheney** [comment]
Closed by #156.
